src/iter/mod.rs

```diff
@@ -20,6 +20,16 @@ pub struct BitIter<T> {
     pub(crate) prefix: [u32; LAYERS - 1],
 }
 
+impl<T: Clone> Clone for BitIter<T> {
+    fn clone(&self) -> Self {
+        BitIter {
+            set: self.set.clone(),
+            masks: self.masks.clone(),
+            prefix: self.prefix.clone(),
```

Comment:
It is not required to explicitly clone masks and prefix, as both implement copy. I may remove this if undesired.